src/Components/Alert/Components/Filters/Price.tsx

```diff
@@ -18,11 +32,70 @@ export const Price: React.FC = () => {
 
   return (
     <>
-      <Text variant="sm-display">Price Range</Text>
+      <Text variant="sm-display" mb={2}>
+        Price Range
+      </Text>
+
       <PriceRange
+        bars={bars}
         priceRange={state.criteria.priceRange || DEFAULT_PRICE_RANGE}
         onDebouncedUpdate={handlePriceRangeUpdate}
       />
     </>
   )
 }
+
+export const PriceFragmentContainer = createFragmentContainer(Price, {
+  aggregations: graphql`
+    fragment Price_aggregations on FilterArtworksConnection {
+      aggregations {
+        slice
+        counts {
+          name
+          value
+          count
+        }
+      }
+    }
+  `,
+})
+
+export const PriceQueryRenderer = () => {
+  const { state } = useAlertContext()
+
+  const artistID = state.criteria.artistIDs?.[0]
+
+  if (!artistID) return <Price />
+
+  return (
+    <SystemQueryRenderer<PriceAggregationsQuery>
+      lazyLoad
+      placeholder={<Price />}
+      variables={{ artistID }}
+      query={graphql`
+        query PriceAggregationsQuery($artistID: String!) {
+          artist(id: $artistID) {
+            filterArtworksConnection(
+              aggregations: [SIMPLE_PRICE_HISTOGRAM]
+              first: 1
+            ) {
+              counts {
+                total
```

Comment:
**question**: Are using this `total` value anywhere? I don't see it used outside of this query.